### PR TITLE
Initial Travis-CI config plus one test

### PR DESCRIPTION
**Pull request by rhyolight**
Added one test. Removed cache setup from constructor.

### CODE REVIEW

**Comment by rhyolight** [comment]
work in progress... checking to ensure tests run in travis...

File: cortipy/cortical_client.py [hunk 1]
```diff
@@ -73,13 +73,7 @@ def __init__(self,
     else:
       self.apiKey = os.environ["CORTICAL_API_KEY"]
     self.apiUrl = baseUrl
-    
-    # Create cache directory if necessary
-    cacheDir = os.path.join(cacheDir, retina)
-    if not os.path.exists(cacheDir):
-      os.makedirs(cacheDir)
```
Comment (rhyolight) [inline cortipy/cortical_client.py:80]: @BoltzmannBrain This was probably copied from the `pycept` constructor, but it is wrong in hindsight. The constructor of an object has a [single responsibility](http://en.wikipedia.org/wiki/Single_responsibility_principle), which is to construct the object. Any other actions being taken in a constructor are misplaced, including making API calls, system calls, file system operations, etc.

The test I added asserts against file system operations, so this has been removed and achieved lazily in the function that writes to the cache.

**Comment by rhyolight** [comment]
@BoltzmannBrain ready.

**Comment by BoltzmannBrain** [comment]
:+1:

File: cortipy/cortical_client.py (hunk 1 above)
Comment (BoltzmannBrain) [inline cortipy/cortical_client.py:80]: Thank you for the explanation :smiley:

**Pull request merged**